log-insights/setup/opentelemetry.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Log Insights: Kubernetes with OpenTelemetry'
+backlink_href: /docs/log-insights/setup
+backlink_title: 'Log Insights: Setup'
+---
+
+If you are running the database in the Kubernetes cluster, you can use the
+[`db_log_otel_server`](/docs/collector/settings#self-managed-servers) setting to
+have the collector listen for OpenTelemetry log data model messages.
+When this is specified, the collector will run the HTTP server behind the scenes
+on the specified port to receive logs at the `/v1/logs` endpoint.
+This should be a local address like `0.0.0.0:4318` and it's recommended to use
+an unprivileged port number (1024 and up) to avoid running the HTTP server as
+root.
+
+**Note:** This is still beta and currently only supporting CloudNativePG logs,
```

Comment:
I guess we don't really say "beta", but I wanted to add some hint that this is still pretty much wip